common/src/main/java/io/novafoundation/nova/common/utils/FlowExt.kt

```diff
@@ -308,6 +308,9 @@ inline fun <T> EditText.bindTo(
             if (text.toString() != inputString) {
                 removeTextChangedListener(textWatcher)
                 setText(inputString)
+                if (hasFocus()) {
+                    setSelection(inputString.length)
+                }
```

Comment:
What if user is editing field in the middle?

Reply:
I guess that nothing will happen. selection will move to end only after input flow value changed.
I think we need to have regression testing

Reply:
Or we can make this logic only for this case

Reply:
But why is this needed in the first place? I thought this is no longer a problem for swap main screen

Reply:
I need this for slippage input field. When I click tips the text will change but selection will not move to the end

Reply:
Lets enable it only by a boolean flag passed to the function to avoid alter default behavior and avoid possible minor regressions